ui/components/ui/nickname-popover/index.scss

```diff
@@ -3,95 +3,66 @@
   &__popover-wrap {
     height: 232px;
     border-radius: 8px;
-    background: #fff;
-    box-shadow: 0 4px 20px rgba(0, 0, 0, 0.214);
+    background: $ui-white;
+    display: flex;
+    justify-content: center;
 
     .popover-header {
       padding: 16px 16px 0 0;
     }
 
     .popover-content {
       margin-top: -15px;
+      align-items: center;
     }
   }
 
-  &__identicon {
-    width: 36px;
-    height: 36px;
-    border-radius: 32px;
-    align-items: center;
-    margin-left: 146px;
-  }
-
   &__address {
-    position: absolute;
-    height: 26px;
-    left: 113px;
-    right: 114px;
-    top: 44px;
-    font-style: normal;
+    @include H4;
```

Comment:
`@include H4` will important the H4 typography styles from `design-system/typography.scss`, so that font-size, font-family, line-height and font-style do not need to be specified